### PR TITLE
x3: Initial crDroid v8.4 Update

### PR DESCRIPTION
**Pull request by Patriot-06**

### CODE REVIEW

**Comment by gwolf2u** [comment]
Should have waited for dt review

**Comment by Patriot-06** [comment]
> Should have waited for dt review

I'm Extremely sorry I didn't knew

File: x3.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "buildtype": "Monthly",
+        "forum": "https://forum.xda-developers.com/t/rom-12-0-rmx2081-crdroid-android-v8-x-unified-official.4432753/",
+        "gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-SL/13-Apr-2022/NikGapps-basic-arm64-12.1-20220413-signed.zip/download",
+        "firmware": "",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "",
+        "paypal": "https://www.paypal.me/patriot235",
+        "telegram": "https://t.me/patriot_235",
+        "dt": "https://github.com/crdroidandroid/android_device_realme_x3",
```
Comment (gwolf2u) [inline x3.json:23]: fix selinux first
https://github.com/crdroidandroid/android_device_realme_x3/blob/12.1/BoardConfig.mk#L96

Comment (Patriot-06) [reply x3.json:23]: Will do

**Pull request merged**